projects/1.16/assets/apotheosis/apotheosis/lang/zh_cn.json

```
@@ -382,6 +382,18 @@
   "enchantment.apotheosis.magic_protection.desc": "减少玩家造成的魔法伤害，同时减少玩家受到的魔法伤害。",
   "enchantment.apotheosis.crescendo": "快速射击",
   "enchantment.apotheosis.crescendo.desc": "增加你的弩在装填完箭后瞄准时的射击次数。",
+  "enchantment.apotheosis.spearfishing": "捕鱼矛",
+  "enchantment.apotheosis.spearfishing.desc": "用投掷的三叉戟杀人可能会抓到一条鱼。",
+  "enchantment.apotheosis.chromatic": "色差",
+  "enchantment.apotheosis.chromatic.desc": "剪下的羊毛会变成随机颜色。",
+  "enchantment.apotheosis.exploitation": "工人剥削",
+  "enchantment.apotheosis.exploitation.desc": "收获的羊毛会加倍，但会使羊受到伤害。",
+  "enchantment.apotheosis.growth_serum": "生长血清",
+  "enchantment.apotheosis.growth_serum.desc": "羊有50%的几率立即长出羊毛。",
+  "enchantment.apotheosis.earths_boon": "地球的恩惠",

```

确定`Earth`是`地球`？

---

我觉得地球的恩惠好听一点，或者改成大地的恩惠或者土地的恩惠？

---

```suggestion
  "enchantment.apotheosis.earths_boon": "大地的恩惠",
```
minecraft里大概没有`地*球*`这个概念